types/terminal-kit/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for terminal-kit 1.26
+// Project: https://github.com/cronvel/terminal-kit#readme
+// Definitions by: katsanva <https://github.com/katsanva>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+import Terminal = require("./Terminal");
+import * as SB from "./ScreenBuffer";
+import * as SBHD from "./ScreenBufferHD";
+import * as R from "./Rect";
+import * as TB from "./TextBuffer";
+
+export as namespace termkit;
+
+export const terminal: Terminal;
+
+export const realTerminal: Terminal;
+
+export function createTerminal(
+  createOptions?: Terminal.CreateOptions
+): Terminal;
+
+export function getParentTerminalInfo(
+  callback: (error: any, codename?: string, name?: string, pid?: number) => void
+): void;
+
+export function getDetectedTerminal(
+  calback: (error: any, term: Terminal) => void
+): void;
+
+export function autoComplete(
+  array: ReadonlyArray<string>,
+  startString: string,
+  returnAlternatives?: boolean,
+  prefix?: string,
+  postfix?: string
+): string;
+
+export function stripEscapeSequences(str: string): string;
+
+export function stringWidth(str: string): number;
+
+export function truncateString(str: string, maxWidth: number): string;
+
+export { default as ScreenBufferHD } from "./ScreenBufferHD";
```

Comment:
Why are you exporting default here? I could be missing something here, but it doesn't look like these files have an `export default` to do that with.

Reply:
Thanks for pointing!

Reply:
Actually, I wouldn't have done it this way... 
It seems like the [underlying library](https://unpkg.com/terminal-kit@1.26.10/lib/ScreenBuffer.js) is not using `export default`, so these .d.ts files should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.